unity-environment/Assets/ML-Agents/Examples/Walker/Scripts/WalkerGroundContact.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class WalkerGroundContact : MonoBehaviour
+{
+    public int index;
+    public WalkerAgent agent;
+    public bool touchingGround;
+    public bool penalizeOnContact;
```

Comment:
Who should be setting this value and when would it be true vs false?

Reply:
Environment creator sets it, from the editor. True for body parts you don't want touching the ground; false for all others.